spec/unit/form_spec.rb

```diff
@@ -116,6 +116,13 @@ def self.name
     end
 
     it 'raises error when attribute is in conflict with form interface' do
+      pending <<-PENDING.strip_heredoc
+      I've updated the builder to _skip_ already existing methods, rather
+      than raising a warning.
+      1) It's needed for allowing input to be called multiple times
+      2) If you've already defined a method, good on ya?
+      DISCUSS
+      PENDING
```

Comment:
The only reason why we define those methods is because of the horrible assumption rails helpers make that the passed in object also exposes attribute values.

I decided to raise an error because if there's an attribute called "save", or "validate" or "attributes" etc. then it will cause weird behavior.